src/sentry/api/endpoints/project_rules_configuration.py

```diff
@@ -33,6 +33,9 @@ def get(self, request: Request, project) -> Response:
             "organizations:integrations-ticket-rules", project.organization
         )
         has_issue_severity_alerts = features.has("projects:first-event-severity-alerting", project)
+        has_latest_adopted_release = features.has(
+            "organizations:latest-adopted-release-filter", project
```

Comment:
@wedamija shouldn't you pass in `project.organization`?

Reply:
Oof, that's what I get for copy/pasting feature logic...